docs/applications/development/clojure-deployment-with-wildfly-ubuntu-14-04.md

```diff
@@ -0,0 +1,241 @@
+description: 'Clojure application deployment with Immutant and WildFly on Ubuntu 14.04 LTS'
+keywords: 'clojure,luminus,leiningen,immutant,jvm,wildfly,jboss'
+license: '[CC BY-ND 3.0](http://creativecommons.org/licenses/by-nd/3.0/us/)'
+modified: 'Monday, December 21st, 2015'
+modified_by:
+  name: Sergey Pariev
+published: 'Monday, December 14th, 2015'
+title: 'Clojure Application Deployment with Immutant and WildFly on Ubuntu 14.04 LTS'
+contributor:
+  name: Sergey Pariev
+  link: https://twitter.com/spariev
+external_resources:
+  - '[Installing Leiningen](http://leiningen.org/#install)'
+  - '[Installing Oracle JDK 8](https://launchpad.net/~webupd8team/+archive/java)'
+  - '[Luminus Framework](http://www.luminusweb.net/docs)'
+  - '[Immutant 2](http://immutant.org/documentation/current/apidoc/)'
+  - '[Script to install JBoss Wildfly 10.x as service in Linux](https://gist.github.com/sukharevd/6087988)'
+
+---
+
+Clojure is a general-purpose programming language with an emphasis on functional programming. It is a dialect of the Lisp programming language running on the Java Virtual Machine (JVM). While Clojure allows you to write elegant and concise code, its ability to make use of the existing JVM infrastructure, such as libraries, tools and application servers, makes it also a very practical choice.
+
+This guide will show how to deploy Clojure application to WildFly - widely used open-source Java application server, developed by RedHat. To simplify the deployment process, suite of libraries, called *Immutant*, will be used.
+
+## Before You Begin
+
+1.  Familiarize yourself with our [Getting Started](/docs/getting-started) guide and complete the steps for setting your Linode's hostname and timezone.
+
+2.  This guide will use `sudo` wherever possible. Complete the sections of our [Securing Your Server](/docs/security/securing-your-server) to create a standard user account, harden SSH access and remove unnecessary network services.
+
+3.  Update your system.
+
+        sudo apt-get update && sudo apt-get upgrade
+
+{: .note}
+>
+>This guide is written for a non-root user. Commands that require elevated privileges are prefixed with `sudo`. If you’re not familiar with the `sudo` command, you can check our [Users and Groups](/docs/tools-reference/linux-users-and-groups) guide.
+
+{: .note}
+>
+>In this guide `example.com` will be used as a domain name, and `linode-user` as a name of non-root user. Substitute your own FQDN and username accordingly.
+
+
+## Install Oracle JDK 8
+
+1.  Add Oracle Java 8 Installer PPA repository to the system:
+
+		sudo add-apt-repository ppa:webupd8team/java
+
+2.  Update package list and install Oracle JDK 8:
+
+		sudo apt-get update
+		sudo apt-get install oracle-java8-installer
+
+	You will be asked twice to agree with "Oracle Binary Code License Agreement for the Java SE Platform Products and JavaFX" by installer. Press &lt;Ok&gt; button to agree the first time, and &lt;Yes&gt; button the second time.
```

Comment:
For the copy editor: &lt; and &gt; translate into < & >, and should be replaced anyway since we use **bold** to show buttons, not `<>` brackets.